recipes/hwdata/all/test_package/conanfile.py

```diff
@@ -0,0 +1,36 @@
+from conan import ConanFile
```

Comment:
@jwillikers I would suggest simplifying this test package to only check if those files are installed. The hwdata package only contains "text" files, nothing to be linked actually, so you could check only if are installed and it should be enough 😸